### DISCUSSION TITLE
"Network error: connection timeout" --PuTTY (inactive)

### DISCUSSION THREAD

**Discussion by ias052** [l3atbc-datadog/pushkin#86] [General] [answered]
I followed the direction and created AWS and installed PuTTY. Now I am running my AWS Instance (green dot, "running"), I loaded the session on PuTTY, but PuTTY gives me "Network error: connection timeout" message (PuTTY "inactive"). What is wrong?

**Comment by restmegan** [comment, marked as answer]
This can happen if internet is slow or loading takes longer than expected. Try closing PuTTY and reopening it, you will have to go through the steps of connecting your EC2 instance again unless you saved your settings

**Reply by restmegan** [reply]
Let me know if this works for you!

**Reply by jkhartshorne** [reply]
@ias052 did that work?

**Reply by jkhartshorne** [reply]
I'm going to close this issue. But if you get a chance, @ias052, please let us know if you were able to get this to work or not. Thanks!

**Comment by restmegan** [comment]
Hi @ias052 ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
